be/src/olap/rowset/segment_v2/column_zone_map.cpp

```diff
@@ -0,0 +1,43 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+#include "olap/rowset/segment_v2/column_zone_map.h"
+
+namespace doris {
+namespace segment_v2 {
+
+Status ColumnZoneMap::load() {
+    DCHECK_GE(_data.size, 4) << "block size must greate than header";
+    _num_pages = decode_fixed32_le((const uint8_t*)_data.data);
+    _data.remove_prefix(4);
```

Comment:
use a temporary Slice to parse, keep input data not changed

Reply:
ok